lib/settings.js

```diff
@@ -5,9 +5,9 @@ const ENV =
 
 export const LOCALE = ENV.LOCALE
 export const GITHUB_ORG = ENV.GITHUB_ORG
-export const API_BASE_URL = ENV.API_BASE_URL
+export const API_URL = ENV.API_URL
```

Comment:
Would reflect environment variable naming improvement in `.env.example`

Reply:
That's right, thank you!